plugins/woocommerce-admin/client/products/sections/images-section.tsx

```diff
@@ -161,43 +174,89 @@ export const ImagesSection: React.FC = () => {
 							</CardBody>
 						) : (
 							<CardBody>
-								<MediaUploader
-									onError={ () => null }
-									onSelect={ ( file ) => {
-										if (
-											images.find(
-												( img ) => file.id === img.id
-											) === undefined
-										) {
-											setValue( 'images', [
-												...images,
-												file,
-											] );
-										}
-									} }
-									onUpload={ ( files ) => {
-										if ( files[ 0 ].id ) {
-											setValue( 'images', [
-												...images,
-												...files,
-											] );
-										}
+								<FormFileUpload
```

Comment:
I'm good with this here for now, but just curious if we want all media upload components to have the same behavior with file uploading (cc @jarekmorawski).

If so, we could move this into the `MediaUploader` component.  But we can save that for a follow-up if we need to do that.

Reply:
Yes. We should use the same interactions across all media upload components. 👍

Reply:
> we could move this into the MediaUploader component. But we can save that for a follow-up if we need to do that.

Good idea. @joshuatf since this PR has been open for so long, I would add that change in a follow-up PR.